src/pybind/mgr/dashboard/frontend/src/app/ceph/pool/pool-form/pool-form.component.ts

```diff
@@ -136,7 +136,14 @@ export class PoolFormComponent implements OnInit {
           validators: [Validators.required, Validators.min(1)]
         }),
         ecOverwrites: new FormControl(false),
-        compression: compressionForm
+        compression: compressionForm,
+        max_bytes: new FormControl(0, {
+          updateOn: 'blur'
```

Comment:
I thinks this is already the default and can be removed.

Reply:
seems `updateOn` is default to `change`?
https://angular.io/api/forms/AbstractControl#updateOn

Reply:
I had only tested how the input was working and the value was only being updated on blur.

But since `cdDimlessBinary` still works without changing this setting and we don't have any `valueChanges` linked to both this `FormControls` I would suggest to try to keep the changes to a minimum and still remove this.

Hope I'm not missing anything, let me know what you think.

Reply:
Thanks!
will remove, no valueChanges are linked to these two control.